GeometricSeqPhenotype.java

```diff
@@ -0,0 +1,316 @@
+import java.util.Arrays;
+
+/**
+ * <b>GeometricSeqPhenotype</b> stores a Virus's genetic sequence and antigenic phenotype,
+ * and tracks the cumulative number of mutations in epitope and non-epitope sites.
+ * Antigenic phenotype is defined as a position in 2D space, where the antigenic distance
+ * between two viruses can be computed as the Euclidean distance between them.
+ * Multiple Viruses can reference a single GeometricSeqPhenotype which is identified by its data contents.
+ *
+ * @author Thien Tran
+ */
+public class GeometricSeqPhenotype extends GeometricPhenotype {
+    /**
+     * The nucleotide sequence of this GeometricSeqPhenotype
+     */
+    private char[] nucleotideSequence;
+
+    /**
+     * The first parameter that determines the position of this GeometricPhenotype in Euclidean space
+     */
+    private double traitA;
+
+    /**
+     * The second parameter that determines the position of this GeometricPhenotype in Euclidean space
+     */
+    private double traitB;
+
+    /**
+     * The number of epitope mutations this GeometricPhenotype went through (counting from the startingSequence)
+     */
+    private int eMutation;
```

Comment:
Is there a standard in this codebase for how counts are named? This is not obviously a count. I would call this epitopeMutationCount, if it was my code.